src/main/kotlin/com/bugsnag/android/gradle/BugsnagPlugin.kt

```diff
@@ -566,13 +569,43 @@ class BugsnagPlugin : Plugin<Project> {
         }
     }
 
-
-    internal fun isNdkUploadEnabled(bugsnag: BugsnagPluginExtension,
-        android: AppExtension): Boolean {
+    internal fun isNdkUploadEnabled(
+        bugsnag: BugsnagPluginExtension,
+        android: AppExtension
+    ): Boolean {
         val usesCmake = android.externalNativeBuild.cmake.path != null
         val usesNdkBuild = android.externalNativeBuild.ndkBuild.path != null
         val default = usesCmake || usesNdkBuild
-        return bugsnag.uploadNdkMappings.getOrElse(default)
+        val unityEnabled = isUnityLibraryUploadEnabled(bugsnag, android)
+        return bugsnag.uploadNdkMappings.getOrElse(default) || unityEnabled
```

Comment:
Is it intended for this to still be `true` in unity projects if `uploadNdkMappings` is `false`? I think this should be

```kotlin
val default = usesCmake || usesNdkBuild || isUnityLibraryUploadEnabled(bugsnag, android)
return bugsnag.uploadNdkMappings.getOrElse(default)
```

Reply:
Nice catch, it should indeed be this.